### DISCUSSION TITLE
brownie import module error

### DISCUSSION THREAD

**Discussion by pgraciae** [smartcontractkit/full-blockchain-solidity-course-py#352] [Q&A] [answered]
Hello!
I have a strange problem with brownie that I have been trying to solve for the last 4 hours without any success and I would really appreciate help :)

In lesson 5 I installed brownie with pipx and followed the course properly, but as soon as I started lesson 6 and executed the first python script (with the file FundMe.sol already compiled) I get the following error:
![imatge](https://user-images.githubusercontent.com/72271001/141646948-151eb2c1-d4ad-43fa-8cee-6467fe8ba701.png)

(In lesson 5 the import worked properly)

When I try to solve it by downloading brownie via pip then I get the following error:

![imatge](https://user-images.githubusercontent.com/72271001/141646998-16c70b86-5a0f-49c5-a7d5-73d5e25f8063.png)

I have tried reinstalling everything but everytime I get the same errors, even in lesson 5 when it worked properly.

Thank you!

PS: I work with ubuntu 20.04 and brownie as a commandline works perfectly, the problem is with python.

**Comment by cromewar** [comment]
Hey @pgraciae it seems you have conflict with python instances, I strongly suggest you to use `virtualenv` following [this](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/) guide.
The steps to follow are: 

1. install virtualenv via pip
2. create a new virtualenv
3. activate it
4. install brownie on it.

Note: also don't forget to select the correct python interpreter on vscode (bottom left corner) and activate the virtualenv also on vscode terminal (same command).

Best regards, Cromewar.

**Reply by pgraciae** [reply]
I already tried and still the same problem... Thank you very much anyway!

**Reply by cromewar** [reply]
Maybe something is messing up your ~/.bashrc config, let's try something else.

1. could you re-run the project on section 5 and see if it still working?
2. Remember Ubuntu used to come with bot python 2.7 and python 3.0 could you try to run using this: `python3 -m brownie` ?

**Reply by pgraciae** [reply]
Section 5 not working now...

![imatge](https://user-images.githubusercontent.com/72271001/141657037-fd6db95c-0dd7-4f8a-a719-57327a919cbf.png)

Not working either...

**Reply by cromewar** [reply]
yes now I'm sure something is messing up with your python installation, try this:

1. `python -m pip install --upgrade eth-brownie`.
2. try to use miniconda python: https://docs.conda.io/en/latest/miniconda.html

**Reply by pgraciae** [reply]
Hi! I finally manage to solve it just by runing the script with brownie:
`$brownie run scripts/deploy.py`
Thank you very much @cromewar for you time!!

**Comment by pgraciae** [comment, marked as answer]
I finally manage to solved it just by running the script with brownie:
$brownie run scripts/deploy.py
